testing/testcafe/tests/pivotGrid/export/onExportingOption.ts

```diff
@@ -0,0 +1,33 @@
+import { ClientFunction } from 'testcafe';
+import createWidget from '../../../helpers/createWidget';
+import url from '../../../helpers/getPageUrl';
+import PivotGrid from '../../../model/pivotGrid';
+
+fixture.disablePageReloads`PivotGrid_fieldChooser`
+  .page(url(__dirname, '../../container.html'));
+
+test('Should call \'onExporting\' when export button clicked', async (t) => {
+  const pivotGrid = new PivotGrid('#container');
+  const exportBtn = pivotGrid.getExportButton();
+
+  await t.click(exportBtn);
+  // eslint-disable-next-line no-underscore-dangle
+  const exportCalled = await ClientFunction(() => (window as any).__exportCalled as boolean)();
+
+  await t.expect(exportCalled).ok();
+}).before(async () => createWidget('dxPivotGrid', {
+  dataSource: {
+    fields: [{
+      caption: 'data A',
+      dataField: 'data_A',
+    }],
+    store: [],
+  },
+  export: {
+    enabled: true,
+  },
+  onExporting() {
```

Comment:
Let's check incomming arguments

Reply:
Let's keep it as is (check only that ```onExporting``` was called)

Because this method doesn't have any specific arguments for export (it only passed reference of the pivot grid widget to the export tools)
[Link to the docs](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#onExporting)